cmd/agentctl2/cmd/cli_cmd.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ligato/vpp-agent/cmd/agentctl2/restapi"
+	"github.com/spf13/cobra"
+)
+
+// RootCmd represents the base command when called without any subcommands.
+var cliConfig = &cobra.Command{
+	Use:     "cli",
```

Comment:
Using `vppcli` might make more sense here. 

Maybe aslo, without an alias, so it's always obvious what it does, even in some scripts.

```sh
agentctl c show interfaces
# vs.
agentctl vppctl show interfaces
```